packages/rum-core/src/domain/view/viewCollection.spec.ts

```diff
@@ -63,20 +69,8 @@ const VIEW: ViewEvent = {
   sessionIsActive: true,
 }
 
-const baseFeatureFlagContexts: FeatureFlagContexts = {
-  findFeatureFlagEvaluations: () => undefined,
-  addFeatureFlagEvaluation: noop,
-  stop: noop,
-}
-const baseConfiguration: RumConfiguration = {
-  ...validateAndBuildRumConfiguration({
-    clientToken: 'xxx',
-    applicationId: 'FAKE_APP_ID',
-    trackResources: true,
-    trackLongTasks: true,
-  })!,
-  ...SPEC_ENDPOINTS,
-}
+const baseFeatureFlagContexts = mockFeatureFlagContexts()
+const baseConfiguration = mockRumConfiguration()
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** I'm wondering if could get rid of these objects and do something like:
```ts
  function setupViewCollection(partialConfiguration, partialFeatureFlagContexts) {
    const collectionResult = startViewCollection(
     mockRumConfiguration(partialConfiguration),
     mockFeatureFlagContexts(partialFeatureFlagContexts),
     ...
```